test/utils/test-utils.js

```diff
@@ -80,6 +82,35 @@ function runAndGetWatchProc(testCase, args = [], setOutput = true) {
     return webpackProc;
 }
 
+const runInitWithAnswers = async (location, answers) => {
+    const runner = runAndGetWatchProc(location, ['init'], false);
+    runner.stdin.setDefaultEncoding('utf-8');
+
+    // Simulate answers by sending the answers after waiting for 1s
```

Comment:
It seems the wait time is `2s`, why not update the comment accordingly?

Reply:
Did that in latest commit. Thanks!